res/css/structures/_AutoHideScrollbar.pcss

```diff
@@ -25,7 +25,7 @@ html {
 }
 
 ::-webkit-scrollbar {
-    width: 6px;
+    width: 12px;
```

Comment:
Won't this make it very different between webkit and non-webkit browsers?

Reply:
\+ @akshattchhabra can you send screenshots before and after the changes? We should verify, that the scrollbars don't overlap with other content or break the layout. Someone should test the final solution with a non-webkit browser.

Reply:
Before:

![image](https://user-images.githubusercontent.com/72541276/217236904-62cbdc99-21d6-4d0f-9788-c5a487dd4392.png)

After:

![image](https://user-images.githubusercontent.com/72541276/217236934-66972d02-752c-403f-ae03-273b65abb2be.png)

Here, I have attached the screenshots.

So, I checked the solution on a non-webkit browser. Yeah, you are right there would be differences between them and I checked about the possible solutions to make this compatible.

Solution:
To resolve this issue, we need to change the "scrollbar-width: thin" to "scrollbar-width: auto". This will work for the non-webkit browsers. Also, we need to completely remove the webkit-scrollbar css. This will ensure that the scrollbars are wide enough and identical for both kinds of browsers. Please find the attached screenshots for solution.

Webkit browser

![image](https://user-images.githubusercontent.com/72541276/217255994-6fe3ba0c-0d06-4e48-8c58-53b631a5f149.png)

non-webkit browser
![image](https://user-images.githubusercontent.com/72541276/217256052-cf34ae23-b4d9-4b19-b8e3-53261d0b2186.png)

This is what I think should be the solution. However, I am not sure if this is correct. Please suggest if you have any other solution.